docs/analytics_tools/python_libraries.md

```diff
@@ -179,8 +179,10 @@ While most Python packages an analyst uses come in JupyterHub, there may be addi
 * Change directory into the project task's subfolder and add `requirements.txt` and/or `conda-requirements.txt`
 * Run `pip install -r requirements.txt` and/or `conda install --yes -c conda-forge --file conda-requirements.txt`
 
-### calitp-data-infra
+(appendix)=
 
-The calitp-data-infra package, used primarily by warehouse mainainers and data pipeline developers, includes utilities that analysts may not often need to interact with, but should be aware of. For instance, the `get_secret_by_name()` and `get_secrets_by_label()` functions in [the package's `auth` module](https://github.com/cal-itp/data-infra/blob/main/packages/calitp-data-infra/calitp_data_infra/auth.py) are used to interact with Google's [Secret Manager](https://console.cloud.google.com/security/secret-manager), the service that securely stores API keys and other sensitive information that underpins many of our data syncs.
+## Appdendix: calitp-data-infra
```

Comment:
nit: Appendix

Reply:
Resolved in [a5a6f7f](https://github.com/cal-itp/data-infra/pull/2856/commits/a5a6f7f2018ff99d7383b40b0ab499b66a34b866)